### PR TITLE
Document procedure to resize OCP4 worker nodes on cloudscale.ch

### PR DESCRIPTION
**Pull request by TheBigLee**
Signed-off-by: Nicolas Bigler <nicolas.bigler@vshn.ch>

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+The following CLI utilities need to be available locally:
+
+* `kubectl`
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+* `oc`
+
+== Increase node disk via terraform
+
+. In the tenant repo set the parameter `openshift4_terraform.terraform_variables.worker_volume_size_gb` to the desired value.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc:28]:
```suggestion
. In the tenant repo, set the parameter `openshift4_terraform.terraform_variables.worker_volume_size_gb` to the desired value.
```

File: docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+. Run commodore to compile the terraform manifests (should be done automatically via ci pipeline)
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc:29]:
```suggestion
. Run Commodore to compile the Terraform manifests (should be done automatically via CI pipeline)
```

File: docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+. Run terraform on the updated terraform manifests (should be done automatically via ci pipeline, the apply stage may be triggered manually)
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc:30]:
```suggestion
. Run Terraform on the updated Terraform manifests (should be done automatically via CI pipeline, the apply stage may be triggered manually)
```

File: docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+
+== Increase the partition and filesystem on the worker nodes
+
+Once the terraform pipeline ran through, the worker nodes should have the new disk size. However the partition and filesystem on the worker nodes need to be resized manually:
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/cloudscale/increase-worker-node-disk.adoc:35]:
```suggestion
Once the Terraform pipeline ran through, the worker nodes should have the new disk size.
However the partition and filesystem on the worker nodes need to be resized manually:
```

**Review by simu — CHANGES_REQUESTED** [review]
LGTM overall. As always, please keep documentation to one sentence per line and use capitalization.

Please make sure that the new page is added in the navigation (in `docs/modules/ROOT/partials/nav-howtos.adoc`)

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**